requirements.txt

```diff
@@ -15,3 +15,4 @@ pytest >= 3.7.4
 pytest-cov >= 2.4.0
 flake8 >= 3.6.0
 flake8_docstrings >= 1.3.0
+pydocstyle == 2.1
```

Comment:
Why?

Reply:
This is for the cron CI. I have tested in the master branch.